Source/WebKit/GPUProcess/graphics/RemoteDisplayListRecorder.cpp

```diff
@@ -75,7 +75,7 @@ ControlFactory& RemoteDisplayListRecorder::controlFactory()
 
 RefPtr<ImageBuffer> RemoteDisplayListRecorder::imageBuffer(RenderingResourceIdentifier identifier) const
 {
-    return m_renderingBackend->imageBuffer(identifier);
+    return RefPtr { m_renderingBackend }->imageBuffer(identifier);
```

Comment:
Why not add `protectedRenderingBackend()` instead?

Reply:
This was the only place in this file that we're ref'ing `m_renderingBackend` so I thought it best to just ref it like this rather than add a protected function for just one single call site. Would we prefer to add a protected function instead?

Reply:
I think we generally prefer having a protected function even if it only appears once.

Reply:
I don't have strong feelings but Rupin was following the pattern I recommended and what I've been doing myself. I personally only add protectedFoo() functions there is more than 1 call to it, so save me some time.

Reply:
I see. I don't have a strong preference either way.